Code/Tools/SceneAPI/SDKWrapper/AssImpSceneWrapper.cpp

```diff
@@ -106,7 +109,11 @@ namespace AZ
 
         void AssImpSceneWrapper::Clear()
         {
-            m_importer.FreeScene();
+            if(m_importer)
+            {
+                m_importer->FreeScene();
+                m_importer = AZStd::make_unique<Assimp::Importer>();
```

Comment:
So you are allocating a new Assimp::Importer in Clear? Why

Reply:
This is wrapping a 3rd party pointer that has likely cached something. Since we are trying to reduce memory load between scene builds this is a place that should be reset. Also there is no way to reset AssImp import flags so in the future it will be possible to customize the flags the Importer uses to post process importing scene files (other than FBX files).